docker-compose.yml

```diff
@@ -178,6 +178,13 @@ services:
     depends_on:
       - external_redis
 
+  elasticsearch:
+    build:
+      context: ./elasticsearch/2.4/
```

Comment:
I'd recommend dropping this 2.4 off. If we want different versions, we'll template it the same way the mongodb and mysql ones are done